src/raftstore/store/peer.rs

```diff
@@ -513,7 +513,10 @@ impl Peer {
         // The leader can write to disk and replicate to the followers concurrently
         // For more details, check raft thesis 10.2.1
         if self.is_leader() {
-            try!(self.send(trans, ready.messages.drain(..), &mut metrics.message));
+            self.send(trans, ready.messages.drain(..), &mut metrics.message).unwrap_or_else(|e| {
+                // We don't care that the message is sent failed, so here just log this error.
+                info!("{} leader send messages err {:?}", self.tag, e);
```

Comment:
I don't think it's the log level that makes user concerned but the message itself. If I see a lot of error message logged at info level, I will consider it's a log level bug.